src-docs/src/views/card/card.js

```diff
@@ -18,14 +18,31 @@ import {
   OuiFlexItem,
 } from '../../../../src/components';
 
-const icons = ['Beats', 'Cloud', 'Logging', 'Kibana'];
+const cards = {
+  Beats: {
+    title: 'Dashboards',
+    iconType: 'dashboardApp',
+  },
+  Cloud: {
+    title: 'Visualize',
+    iconType: 'visualizeApp',
+  },
+  Logging: {
+    title: 'Discover',
+    iconType: 'discoverApp',
+  },
+  Kibana: {
```

Comment:
Feels like some of these var names should be updated to not be so Elastic. Beats and Kibana should def be changed. Maybe update them to something similar to the title?